vmdb/app/controllers/miq_ae_class_controller.rb

```diff
@@ -479,15 +479,13 @@ def build_details_grid(view,mode=true)
     new_column.add_attribute("type", 'ro')
     new_column.text = header
 
-    records = Array.new
```

Comment:
There are a number of places where a variable is initialized to an Array and then later the `.each { ary.push }` pattern comes in.  In those cases, we can use .collect or just use the Array itself and set the variable directly.